File: tsodd-service/src/main/java/ru/thetenzou/tsoddservice/controller/Hello.java
package ru.thetenzou.tsoddservice.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;


@RestController
@RequestMapping("/api/tsodd/v1")
public class Hello {
    
    @GetMapping(value="/helloWorld")
    public String hello() {
        return new String("hello world");
    }

    @GetMapping(value="/helloUser")
    public String helloUser() {
        return new String("hello user");
    }

    @GetMapping(value="/helloAdmin")
    public String helloAdmin() {
        return new String("hello admin");
    }
    
}

File: tsodd-service/src/main/kotlin/ru/thetenzou/tsoddservice/common/controller/Hello.kt
package ru.thetenzou.tsoddservice.common.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController


@RestController
@RequestMapping("/api/tsodd/v1")
class Hello {
    @GetMapping("/helloWorld")
    fun hello(): String {
        return "hello world"
    }

    @GetMapping("/helloUser")
    fun helloUser(): String {
        return "hello user"
    }

    @GetMapping("/helloAdmin")
    fun helloAdmin(): String {
        return "hello admin"
    }
}
